src/components/activity/ActivityFormPanel.svelte

```diff
@@ -458,31 +493,42 @@
         <details open style:cursor="pointer">
           <summary>Sequencing</summary>
 
-          <div class="mt-2 mb-3">
-            <label for="simulationDatasetId">Simulation Dataset ID</label>
-            <input class="st-input w-100" disabled name="simulationDatasetId" value={$simulationDatasetId ?? 'None'} />
+          <div class="p-2">
+            <Input layout="inline">
+              <label use:tooltip={{ content: 'Simulation Dataset ID', placement: 'left' }} for="simulationDatasetId"
+                >Simulation Dataset ID</label
```

Comment:
This auto-format got a little wonky. Can we format each on a newline?
```svelte
<label use:tooltip={{ content: 'Simulation Dataset ID', placement: 'left' }} for="simulationDatasetId">
  Simulation Dataset ID
</label>
```

Reply:
Let me try, it kept auto formatting back to the wonky state.

Reply:
Alright, worked around it by pushing the tooltip props to a new line which made things nicer for these long label lines

Reply:
O weird in my editor if I format it to the example above it keeps that format